### PR TITLE
Use wasabi flavour for the quicklogin module

### LINKED ISSUE #6139: Use `wasabi` variant for running `quicklogin`
**Current behaviour**
Running `quicklogin` script installs `jalapeno` variant which is more CI-centric (source: p1648716318570869-slack-CGPNUU63E). 

**Expected behaviour**
As `wasabi` is a developer build, we should run `quicklogin` on it.

@kidinov - does it sound good to you?

### PR DESCRIPTION
**Pull request by kidinov**
Closes: #6139 

### Description

Replaced Jalapeno to Wasapi as a build flavour for the quicklogin module

### Testing instructions

Look into `woo_login.sh` and follow the instruction

### CODE REVIEW

File: quicklogin/woo_login.sh [hunk 1]
```diff
@@ -10,12 +10,14 @@ else
   adbPath='~/Library/Android/sdk/platform-tools/adb'
 fi
 
-eval $adbPath shell pm clear com.woocommerce.android.prealpha
+packageName="com.woocommerce.android.dev"
 
-./gradlew installJalapenoDebug
+eval $adbPath shell pm clear $packageName
+
+./gradlew installWasabiDebug
```
Comment (wzieba) [inline quicklogin/woo_login.sh:17]:
```suggestion
```

Removing this line will speed up the build as `:quicklogin:installDebug` already installs the app (so we'll save on checking for up-to-date tasks).

**Review by wzieba — APPROVED** [review]
Looks good, thanks!

File: quicklogin/woo_login.sh (hunk 1 above)
Comment (kidinov) [reply quicklogin/woo_login.sh:17]: I just tried this without app installed and it failed with:

```
onError: commandError=true message=INSTRUMENTATION_FAILED: com.woocommerce.android.dev.quicklogin/androidx.test.runner.AndroidJUnitRunner
android.util.AndroidException: INSTRUMENTATION_FAILED: com.woocommerce.android.dev.quicklogin/androidx.test.runner.AndroidJUnitRunner
INSTRUMENTATION_STATUS: Error=Unable to find instrumentation target package: com.woocommerce.android.dev
INSTRUMENTATION_STATUS: id=ActivityManagerService
INSTRUMENTATION_STATUS_CODE: -1
        at com.android.commands.am.Instrument.run(Instrument.java:524)
        at com.android.commands.am.Am.runInstrument(Am.java:199)
        at com.android.commands.am.Am.onRun(Am.java:80)
        at com.android.internal.os.BaseCommand.run(BaseCommand.java:62)
        at com.android.commands.am.Am.main(Am.java:50)
        at com.android.internal.os.RuntimeInit.nativeFinishInit(Native Method)
        at com.android.internal.os.RuntimeInit.main(RuntimeInit.java:374)
Starting: Intent { cmp=com.woocommerce.android.dev/com.woocommerce.android.ui.main.MainActivity }
Error type 3
Error: Activity class {com.woocommerce.android.dev/com.woocommerce.android.ui.main.MainActivity} does not exist.
```

@wzieba 
Could it be that you tested it with installed app?

Comment (wzieba) [reply quicklogin/woo_login.sh:17]: Yes, sorry @kidinov my bad. I just tested it and `:quicklogin:installDebug` doesn't install the app (it only assembles it).

**Pull request merged**